techniques/system/distributePolicy/1.0/rsyslogConf.st

```diff
@@ -24,6 +24,8 @@ bundle agent install_rsyslogd {
 
     root_server|role_rudder_relay_top::
       "rsyslog_source_file" string => "rudder-rsyslog-root.conf";
+      "rudder-reports_db_name" string => "&RUDDER_REPORTS_DB_NAME&";
+      "rudder-reports_db_user" string => "&RUDDER_REPORTS_DB_USER&";
```

Comment:
This variable name has a dash in it - use an underscore instead please, that's not a valid CFEngine name